packages/beeq/.storybook/main.ts

```diff
@@ -13,10 +16,10 @@ export default {
   },
   stories: ['../src/_storybook/**/*.mdx', '../src/**/*.stories.@(ts|tsx)'],
   addons: [
-    '@storybook/addon-essentials',
-    '@storybook/addon-a11y',
-    '@beeq/storybook-addon-html',
-    '@chromatic-com/storybook',
+    getAbsolutePath('@storybook/addon-docs'),
+    getAbsolutePath('@storybook/addon-a11y'),
+    // getAbsolutePath('@beeq/storybook-addon-html'),
```

Comment:
I'm temporarily commenting this out while I update the Storybook HTML addon to be compatible with SB v9.